web/src/hotkey.js

```diff
@@ -113,6 +113,10 @@ const keydown_cmd_or_ctrl_mappings = {
     190: {name: "narrow_to_compose_target", message_view_only: true}, // '.'
 };
 
+const keydown_cmd_or_ctrl_with_shift_mappings = {
+    80: {name: "toggle_edit_preview", message_view_only: true}, // 'P'
```

Comment:
I am not sure but `toggle_edit_preview` seems related to message edit UI. Would something like `toggle_compose_preview` or `toggle_message_preview` be better?

Also, do we need this shortcut while editing this message as well?

Reply:
Sure, I renamed it to `toggle_compose_preview`.

Yes, while editing would be nice too, but could that be handled in a future pull request? I took a swing at it but wasn't able to come up with a solution quickly.